tools/lsp/util.rs

```diff
@@ -169,3 +203,13 @@ fn to_lsp_diag_level(level: DiagnosticLevel) -> lsp_types::DiagnosticSeverity {
         _ => lsp_types::DiagnosticSeverity::INFORMATION,
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_invalid_url() {
```

Comment:
(I prefer if the test is just after the function it tests, also the test module make the test name longer with little value)